src/libraries/System.Private.Uri/src/System/Uri.cs

```diff
@@ -114,10 +116,22 @@ private enum Flags : ulong
             FragmentIriCanonical = 0x40000000000,
             IriCanonical = 0x78000000000,
             UnixPath = 0x100000000000,
+
+            CustomParser_ParseMinimalAlreadyCalled = 1UL << 62,
```

Comment:
It'd be helpful to add an XML comment about what this is.  Ideally we'd do so for the rest of the values, too, but that needn't be in this PR.